src/main/java/core/basesyntax/service/impl/FruitServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.OperationWithFruit;
+import core.basesyntax.model.TransactionInfo;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.Operation;
+import java.util.List;
+
+public class FruitServiceImpl implements FruitService {
+    private final Operation operationService;
```

Comment:
```suggestion
    private final OperationHandler operationService;
```
краще все ж змінити назву перед переносом проекту при виконанні наступного завдання по написанню тестів